style/sass/.scss-lint.yml

```diff
@@ -12,7 +12,6 @@ linters:
 
   BemDepth:
     enabled: false
-    max_elements: 1
```

Comment:
I hear ya…but this linter is disabled any way, I just saw no reason to define an option that isn't even used.